src/main/java/by/itech/lab/supplier/domain/User.java

```diff
@@ -1,20 +1,15 @@
 package by.itech.lab.supplier.domain;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-import org.springframework.security.core.GrantedAuthority;
-import org.springframework.security.core.userdetails.UserDetails;
+import lombok.*;
```

Comment:
![image](https://user-images.githubusercontent.com/44436165/96725636-b9157200-13b9-11eb-9a5e-5d4e9727df0f.png)
you must be kidding